target/linux/rtl838x/base-files/etc/board.d/02_network

```diff
@@ -57,7 +57,7 @@ rtl838x_setup_switch()
 	done
 	json_select ..
 	lan_list=$(echo $lan_list | xargs -n1 | sort -V | xargs)
-	ucidef_set_interface_lan "$lan_list"
+	ucidef_set_interface_lan "$lan_list" dhcp
 }
```

Comment:
this seems to be unexpected behavior. I'd expect any owrt unit to come up as 192.168.1.1 upon boot. If I want to change this i can log in and change this behaviour.

Reply:
> this seems to be unexpected behavior. I'd expect any owrt unit to come up as 192.168.1.1 upon boot. If I want to change this i can log in and change this behaviour.

@blogic I also have that expectation/general guideline, even if it's a switch or repeater (as sold by the vendor)
However, I think after you merged the PR with `DEVICE_TYPE = basic`, there won't be any DHCP server left to announce the 192.168.1.0/24 network, so we have no choice anymore?

Reply:
@adschm: As already outlined in https://github.com/openwrt/openwrt/pull/3476#issuecomment-702683155, current documentation doesn't seem to imply the presence of a DHCP server. Almost every switch (with vendor firmware) I've seen in my life had a fixed default IP address (and of course no DHCP server), so that's probably the least surprise for many people.

Even though I created this PR with the impression that DHCP client mode could be a more suitable default setting for a non-router device (as sold by the vendor), I'm fine with closing it, too, because it's not a very important issue IMO.

Reply:
My point is entirely different: With https://github.com/openwrt/openwrt/commit/f422488337c695373afd8c411484534e06dd874d, dhcp servers are removed. I was under the impression that no IPv4 access would be available anymore, but it seems that at least the 192.168.1.1 address itself is actually still assigned anyway.